src/detect-engine-analyzer.c

```diff
@@ -1047,6 +1047,16 @@ void EngineAnalysisRules2(const DetectEngineCtx *de_ctx, const Signature *s)
             break;
     }
 
+    if (s->init_data->is_rule_state_dependant) {
+        jb_open_object(ctx.js, "rule_state_dependant");
+        jb_set_uint(ctx.js, "rule_depends_on_sid", s->init_data->rule_state_dependant_id);
+        jb_set_string(ctx.js, "rule_depends_on_flowbit",
```

Comment:
can this just be `flowbit`? Seems the key is a bit redundant with the object name.

Also, as Shivani is asking: this is 1-N essentially, can we list them all? I'm changing my mind a bit on this. I think being complete is probably best.

So
```json
"rule_state_dependant": {
    "sids": [ 1901, 124, 666 ],
    "flowbits": [ "fb2", "abc" ],
},
```